docs/releasing.md

```diff
@@ -19,3 +19,21 @@
 6. Perform the [image promotion process](https://github.com/kubernetes/k8s.io/tree/master/k8s.gcr.io#image-promoter)
 7. Publish release
 8. Email `kubernetes-sig-scheduling@googlegroups.com` to announce the release
+
+## Notes
+See [post-descheduler-push-images dashboard](https://testgrid.k8s.io/sig-scheduling#post-descheduler-push-images) for staging registry image build job status.
+
+List images in staging registry.
+```
+gcloud container images list --repository gcr.io/k8s-staging-descheduler
+```
+
+List descheduler image tags in the staging registry.
+```
+gcloud container images list-tags gcr.io/k8s-staging-descheduler/descheduler
+```
+
+Pull image from the staging registry.
+```
+docker pull gcr.io/k8s-staging-descheduler/descheduler:v20200206-0.9.0-94-ge2a23f284
```

Comment:
curious, is there a latest tag too?

Reply:
There is not a latest tag in the staging registry. I'd have to do some research to find out if it is possible have a latest tag automatically created and updated.

Reply:
Neither the staging and production registries have a latest tag. I also check a few other k8s projects and they also do not have a latest tag.